bundle/orchestrate/dags/meltano.py

```diff
@@ -123,41 +123,42 @@ def _meltano_job_generator(schedules):
         common_tags.append(f"schedule:{schedule['name']}")
         common_tags.append(f"job:{schedule['job']['name']}")
         interval = schedule["cron_interval"]
+        args = DEFAULT_ARGS.copy()
+        args["start_date"] = datetime.utcnow()
 
-        for idx, task in enumerate(schedule["job"]["tasks"]):
-            logger.info(
-                f"Considering task '{task}' of schedule '{schedule['name']}': {schedule}"
-            )
-            args = DEFAULT_ARGS.copy()
-            args["start_date"] = datetime.utcnow()
-            dag_id = f"{base_id}_task{idx}"
-            task_tags = common_tags.copy()
-
-            dag = DAG(
-                dag_id,
-                description=f"Meltano run task[{idx}]: '{task}'",
-                tags=task_tags,
+        with DAG(
+                base_id,
+                tags=common_tags,
                 catchup=False,
                 default_args=args,
                 schedule_interval=interval,
                 max_active_runs=1,
-            )
-
-            if isinstance(task, Iterable) and not isinstance(task, str):
-                run_args = " ".join(task)
-            else:
-                run_args = task
-
-            elt = BashOperator(
-                task_id=f"run_task_{idx}",
-                bash_command=f"cd {PROJECT_ROOT}; {MELTANO_BIN} run {run_args}",
-                dag=dag,
-            )
-
-            globals()[dag_id] = dag
-            logger.info(
-                f"Task DAG created for schedule '{schedule['name']}', task='{run_args}'"
-            )
+        ) as dag:
+            for idx, task in enumerate(schedule["job"]["tasks"]):
+                logger.info(
+                    f"Considering task '{task}' of schedule '{schedule['name']}': {schedule}"
+                )
+
+                task_id = f"{base_id}_task{idx}"
+
+                if isinstance(task, Iterable) and not isinstance(task, str):
+                    run_args = " ".join(task)
+                else:
+                    run_args = task
+
+                run_task = BashOperator(
```

Comment:
```suggestion
                BashOperator(
```

Reply:
Since we're not defining any dependencies yet the variable lights up in my editor because its not used.